manifests/helm-profiles/openshift.yaml

```diff
@@ -16,3 +16,11 @@ pilot:
   cni:
     enabled: true
     provider: "multus"
+containerSecurityContext:
```

Comment:
not sure about this. I believe this profile is used anywhere, not only in gateway deployments

Reply:
Yes, but the `containerSecurityContext` is only referenced in the gateway chart.